sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AADAuthenticationProperties.java

```diff
@@ -695,40 +617,45 @@ public boolean isAllowedGroup(String group) {
                        .contains(group);
     }
 
-    @Override
-    public void afterPropertiesSet() {
-
-        if (!StringUtils.hasText(baseUri)) {
-            baseUri = "https://login.microsoftonline.com/";
-        } else {
-            baseUri = addSlash(baseUri);
+    private void setDefaultValueFromAzureGlobalProperties(AzureGlobalProperties global) {
```

Comment:
There's another way to do this, which will provide the default value before binding. So there's no need to autowire the globalAzureProperties in this configuration properties. 
```java
@ConfigurationProperties(prefix = AADAuthenticationProperties.PREFIX)
@Bean
public AADAuthenticationProperties aadAuthenticationProperties() {
    return loadProperties(this.azureGlobalProperties, new AADAuthenticationProperties());
}
```

Reply:
@saragluna , updated.